docs: Histogram update added missing options (#92617)
grafana/grafana: docs/sources/panels-visualizations/visualizations/histogram/index.md

This will merge all series and fields into a combined histogram.

### Stacking

> Controls how multiple series are displayed in the histogram. The three settings are:
> 
> #### Off
> 
> Series are not stacked, shown side by side.
> 
> #### Normal
> 
> Series are stacked on top of each other, showing cumulative values.
> 
> #### 100%
> 
> Stacks series to fill 100% of the chart, showing the relative proportion of each series.

Reviewer: ```suggestion
Controls how multiple series are displayed in the histogram. Choose from the following:

- **Off** - Series are not stacked, but instead shown side by side.
- **Normal** - Series are stacked on top of each other, showing cumulative values.
- **100%** - Series are stacked to fill 100% of the chart, showing the relative proportion of each series.
```

Gradient display is influenced by the **Fill opacity** setting.

> - **None** - No gradient fill. This is the default setting.

Reviewer: ```suggestion
Choose from the following:

- **None** - No gradient fill. This is the default setting.
```

Gradient color is generated based on the hue of the line color.

> #### Scheme

Reviewer: I know you were just following the pattern here, but let's fix it while you're working on this. Can you please make these **Gradient mode** options a bullet list like what I've suggested for the **Stacking** options.
